snuba/admin/views.py

```diff
@@ -11,3 +16,23 @@ def root() -> Response:
 @application.route("/health")
 def health() -> Response:
     return Response("OK", 200)
+
+
+@application.route("/configs", methods=["GET"])
+def configs() -> Response:
+    config_data = [
+        {"key": k, "value": v, "type": get_config_type(v)}
+        for (k, v) in state.get_raw_configs().items()
+    ]
+
+    return Response(json.dumps(config_data), 200, {"Content-Type": "application/json"},)
+
+
+def get_config_type(value: Union[str, int, float]) -> str:
+    if isinstance(value, str):
+        return "string"
+    if isinstance(value, int):
+        return "int"
+    if isinstance(value, float):
+        return "float"
+    raise ValueError("Unexpected config type")
```

Comment:
I would prefer this be in its own file. I have a feeling this one would get polluted fairly quickly

Reply:
Good point, this should probably move. Dumping it here for now as I'm not sure how it will fit with Nikhar's API changes yet.

Reply:
Actually probably better to add just that module now, moved it over there.